engine/kv_engine.cpp

```diff
@@ -275,6 +276,8 @@ Status KVEngine::RestoreSkiplist(DLDataEntry *pmem_data_entry, DataEntry *) {
     skiplists_.push_back(std::make_shared<Skiplist>(
         (DLDataEntry *)pmem_data_entry, key, id, pmem_allocator_, hash_table_));
     skiplist = skiplists_.back().get();
+    ConcurrentRebuildSorted::entries_offsets_.insert(
```

Comment:
Need to check if opt_restore_sorted is true?

Reply:
Yes，It needs